piperider_cli/assertion_engine/assertion.py

```diff
@@ -37,10 +37,20 @@ def round_trip_load_yaml(file_path):
     return payload
 
 
-def load_yaml_configs(path):
+def load_yaml_configs(path, config_path):
     passed: List[str] = []
     failed: List[str] = []
     content: Dict = {}
+
+    if config_path is not None:
```

Comment:
Personally, I will suggest just prfer to use positive tests in if condition.
It can improve the readability of your code.
ex:
```
if config_path:
  project_config = safe_load_yaml(config_path)
  if project_config:
    passed.append(config_path)
    payload = project_config.get('tables', {})
    always_merger.merge(content, payload)
  else:
    failed.append(confg_path)
```

Reply:
ok, we will refine it,
and `if config_path:` can also cover more cases in this scenario. e.g., `config_path` is `None` or an empty string.